src/slurm_plugin/common.py

```diff
@@ -378,7 +378,7 @@ def get_cluster_instances(self, include_head_node=False, alive_states_only=True)
         if alive_states_only:
             args["Filters"].append({"Name": "instance-state-name", "Values": list(EC2_INSTANCE_ALIVE_STATES)})
         if not include_head_node:
-            args["Filters"].append({"Name": "tag:aws-parallelcluster-node-type", "Values": ["Compute"]})
+            args["Filters"].append({"Name": "tag:parallelcluster:node-type", "Values": ["Compute"]})
```

Comment:
I think we should align other tags too. e.g. tag:ClusterName but we can do it in a separate patch.